pkg/backup/schedule_exec.go

```diff
@@ -574,10 +581,16 @@ func init() {
 					ExecutorPTSMetrics: &pm,
 					RpoMetric: metric.NewGauge(metric.Metadata{
 						Name:        "schedules.BACKUP.last-completed-time",
-						Help:        "The unix timestamp of the most recently completed backup by a schedule specified as maintaining this metric",
+						Help:        "The unix timestamp of the most recently completed tenant backup by a host schedule specified as maintaining this metric",
 						Measurement: "Jobs",
 						Unit:        metric.Unit_TIMESTAMP_SEC,
 					}),
+					RpoTenantMetric: metric.NewExportedGaugeVec(metric.Metadata{
+						Name:        "schedules.BACKUP.tenant-last-completed-time",
```

Comment:
nit: we're trying to scrub all mentions of "tenant" in our codebase and docs. Use virtual cluster instead.

Reply:
how about this metric name: "schedules.BACKUP.last-completed-time-by-virtual_cluster"